diff --git a/src/lgfx/v1/platforms/sdl/Panel_sdl.cpp b/src/lgfx/v1/platforms/sdl/Panel_sdl.cpp
index 89e6cdc..1b4b3ea 100644
--- a/src/lgfx/v1/platforms/sdl/Panel_sdl.cpp
+++ b/src/lgfx/v1/platforms/sdl/Panel_sdl.cpp
@@ -252,7 +252,9 @@ namespace lgfx
     addKeyCodeMapping(SDLK_RIGHT, 37);
     addKeyCodeMapping(SDLK_UP, 36);
 
+#if !defined(__EMSCRIPTEN__)
     SDL_CreateThread((SDL_ThreadFunction)detectDebugger, "dbg", &_inited);
+#endif
 
     _update_in_semaphore = SDL_CreateSemaphore(0);
     _update_out_semaphore = SDL_CreateSemaphore(0);
